### PR TITLE
relative difference comparison

### PR DESCRIPTION
**Pull request by 0ctagon**

### CODE REVIEW

**Comment by 0ctagon** [comment]
Still needs a documentation update, but first lets check if everything is correct.

**Comment by 0ctagon** [comment]
See: 
https://plothist.readthedocs.io/en/relative_difference/_images/1d_comparison_ratio.svg
https://plothist.readthedocs.io/en/relative_difference/_images/1d_comparison_relative_difference.svg
Same points, different uncertainties

**Review by cyrraz — CHANGES_REQUESTED** [review]

**Comment by 0ctagon** [comment]
I don't remember if we want to remove the split option. It is still implemented.

File: plothist/plotters.py [hunk 1]
```diff
@@ -564,15 +570,27 @@ def plot_comparison(
...
 
         if ratio_uncertainty == "split":
             ax.bar(
                 x=hist_2.axes[0].centers,
                 bottom=np.nan_to_num(
-                    1 - h2_scaled_uncertainties, nan=comparison_ylim[0]
+                    bottom_shift - h2_scaled_uncertainties, nan=comparison_ylim[0]
                 ),
                 height=np.nan_to_num(
                     2 * h2_scaled_uncertainties, nan=2 * comparison_ylim[-1]
```
Comment (cyrraz) [inline plothist/plotters.py:596]: `nan = comparison_ylim[-1] - comparison_ylim[0]`

**Review by cyrraz — CHANGES_REQUESTED** [review]

**Comment by 0ctagon** [comment]
Uncertainties are the same now:
https://github.com/cyrraz/plothist/blob/83f46a691c4c6d4cb45ef750c869f1d9218028a8/docs/img/1d_comparison_relative_difference.svg
https://github.com/cyrraz/plothist/blob/83f46a691c4c6d4cb45ef750c869f1d9218028a8/docs/img/1d_comparison_ratio.svg

**Comment by 0ctagon** [comment]
HEP svg files changed because of the error bar display commit, but nothing changed visually.

**Review by cyrraz — APPROVED** [review]

**Pull request merged**